guides/v2.1/coding-standards/docblock-standard-general.md

```diff
@@ -492,21 +496,24 @@ public function deleteDirectory($path)
 #### @return tag
 {:#return}
 
-If there is no explicit return statement in a method or function, a `@return void` should be used in the documentation.
+In general method return type signatures should be prefered over `@return` type annotations.
+If that is not possible due to ambiguous return types, the `@return` type annotation must be used.
+
+If there is no explicit return statement in a method or function, a `void` return type should be declarted in the method signature.
```

Comment:
Hm, this one sounds better, should -> must and please remove the part "If the method does not `return` a value, the method type signature should be `void`, e.g. `function example(): void {}`."